build.gradle

```diff
@@ -160,6 +178,13 @@ project('client') {
         exclude "**/impl/**";
     }
 }
+
+project('plugins') {
```

Comment:
'plugins' needs to be added to 'publishAllJars' so that it can be imported as its own separate artifact by implementors of the auth plugin.